itest/lnd_route_blinding_test.go

```diff
@@ -1510,3 +1511,103 @@ func testBlindedPaymentHTLCReForward(ht *lntest.HarnessTest) {
 		require.Fail(ht, "timeout waiting for sending payment")
 	}
 }
+
+// testGivenBlindedRouteInvoices tests lnd's ability to create a blinded
+// payment path when the blinded path is (partly) pre-specified. Which it then
+// inserted into an invoice, sent to an invoice with a blinded path and
+// forwarded payment in a blinded route and finally, receiving the payment.
```

Comment:
Can you rephrase second sentence of this comment (Which it then inserted...)
maybe just something like this:
```
This test creates the invoice, and makes sure a payment can be sent to it successfully.
```